boskos/README.md

```diff
@@ -0,0 +1,50 @@
+# WIP boskos
+
+
+## Background
+[βοσκός](https://en.wiktionary.org/wiki/%CE%B2%CE%BF%CF%83%CE%BA%CF%8C%CF%82) - cattle-master in greek!
+
+boskos is a http service that allow k8s test jobs to run independently from GCP projects.
+
+## Introduction
+
+Imagine boskos is a ranch with cattle, and to perform each job in the ranch you need to hold a cattle. 
+
+The ranch is init with GCP projects, one per line, from `-config` flag
+
+Interface:
+
+* `/request?job=FOO&duration=X`: with job FOO, checkout a cattle for duration X.
```

Comment:
Units of X? Also, what happens if it doesn't get returned in that time? I think right now boskos will give the project out to some other test, but what if the original keeps mucking about in it? We could consider having boskos supply a service account that it invalidates when the test is returned/timed out?